enclave-tls/src/tls_wrappers/openssl/init.c

```diff
@@ -40,5 +41,7 @@ tls_wrapper_err_t openssl_tls_init(tls_wrapper_ctx_t *ctx)
 
 	ctx->tls_private = ssl_ctx;
 
+	etls_pthread_key_init();
```

Comment:
It is not appropriate to name it with etls_ prefix because this function is not implemented as enclave-tls core functions, even though it has such a potential.

Reply:
It is better to have a prefix such as "per_thread_".